src/protagonist/DLCS.Repository/Messaging/AssetNotificationSender.cs

```diff
@@ -99,13 +98,13 @@ public async Task<bool> SendIngestAssetRequest(Asset assetToIngest, Cancellation
 
     private async Task<bool> SendCleanupAssetRequest(Asset assetToCleanup, CustomerPathElement customerPathElement, CancellationToken cancellationToken = default)
     {
-        var notificationRequest = new
+        var request = new CleanupAssetNotificationRequest()
         {
-            Id = assetToCleanup.Id.ToString(),
+            Asset = assetToCleanup,
             CustomerPathElement = customerPathElement
         };
-
-        return await topicPublisher.PublishToAssetModifiedTopic(JsonConvert.SerializeObject(notificationRequest, Formatting.None, jsonSerializerSettings), ChangeType.Delete, cancellationToken);
+        
+        return await topicPublisher.PublishToAssetModifiedTopic(JsonConvert.SerializeObject(request, Formatting.None, jsonSerializerSettings), ChangeType.Delete, cancellationToken);
```

Comment:
It shouldn't make a difference but as we're using STJ to deserialise these would it make more sense to serialise with STJ too?

Reply:
I initially used newtonsoft as it's what the rest of the API uses, however git is faster, so I've modified this to use STJ